src/deployment/flavors.md

```diff
@@ -143,9 +143,10 @@ section at the end of this document.
 
 ### Plugin configurations
 
-If your app uses a Flutter plugin, you need to update the `ios/Podfile`.
+If your app uses a Flutter plugin, you need to update
+`ios/Podfile` (if developing for iOS) and `macos/Podfile` (if developing for macOS).
 
-1. In `ios/Podfile` change the default for
+1. In `ios/Podfile`/`macos/Podfile` change the default for
```

Comment:
Note for myself/reviewer for when page preview is generated: is this slash readable?

Reply:
This wasn't very readable, so I replaced it with an "and".